Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -1,285 +1,167 @@
-            cell.leftLabel.numberOfLines = 0;
-            cell.leftLabel.lineBreakMode = .byWordWrapping
-            cell.leftLabel.font = UIFont.systemFont(ofSize: 15)
-            cell.rightLabel.font = UIFont.systemFont(ofSize: 14)
-
-            let attributedDesc = NSMutableAttributedString(string: " "+brbAccount.history[indexPath.row].name + "\n " + brbAccount.history[indexPath.row].timestamp, attributes:nil)
-            let newLineLoc = (attributedDesc.string as NSString).range(of: "\n").location
-            if newLineLoc != NSNotFound {
-                attributedDesc.addAttribute(NSAttributedStringKey.font, value: UIFont.systemFont(ofSize: 12), range: NSRange(location: newLineLoc + 1, length: attributedDesc.string.count - newLineLoc - 1))
-                attributedDesc.addAttribute(NSAttributedStringKey.foregroundColor, value: UIColor(white: 0.40, alpha: 1), range: NSRange(location: newLineLoc + 1, length: attributedDesc.string.count - newLineLoc - 1))
-                attributedDesc.addAttribute(NSAttributedStringKey.font, value: UIFont.boldSystemFont(ofSize: 16), range: NSRange(location: 0, length: newLineLoc))
-            }
-
-            cell.leftLabel.attributedText = attributedDesc
-            cell.rightLabel.text = "-$" + brbAccount.history[indexPath.row].amount
-            cell.rightLabel.textColor = .eateryRed
+            
+            loginViewController.view.isHidden = false
+            
+        case (.account, .account), (.login, .login):
+            break
+            
         }
-
-        return cell
-    }
-
-    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        if section == 0 {
-            return UIView()
-        }
-
-        return historyHeader
-    }
-
-}
-
-extension BRBViewController: UITableViewDelegate {
-
-    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        return indexPath.section == 0 ? 48 : indexPath.section == 1 && indexPath.row < brbAccount.history.count ? 64 : tableView.rowHeight
-    }
-
-    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return section == 0 ? 0 : 56.0
+        
+        state = newState
     }
 
 }
 
-extension BRBViewController: BRBLoginViewDelegate {
-
-    func brbLoginViewClickedLogin(brbLoginView: BRBLoginView, netid: String, password: String) {
-        requestStart = Date()
+// MARK: -
 
+extension BRBViewController: BRBLoginViewControllerDelegate {
+    
+    func loginViewController(_ loginViewController: BRBLoginViewController,
+                             didRequestLoginWithNetid netid: String,
+                             password: String) {
         connectionHandler.netid = netid
         connectionHandler.password = password
         connectionHandler.handleLogin()
+        
+        loginViewController.isLoading = true
+        
+        requestStart = Date()
     }
-
+    
```

Comment:
delete whitespace

Reply:
In general should we keep whitespace only lines?

Reply:
its fine to have newlines if it helps with reading code but i like to delete unnecessary whitespace like in this case

Reply:
What about whitespace between lines of code that are not top-level?

```
class A {
    func x() {
        ...
    }
____< whitespace here?
    func y() {
        ...
    }
}
```

Reply:
i would also prefer to not have whitespace there

Reply:
Ok, I've enabled the "Automatically Trim Whitespace only lines" feature in Xcode. There are tons of lines that are whitespace only that I've committed (many many more than are mentioned in PR comments). Instead of going back through and deleting them all individually I'd rather just let them dissipate over time as the files get edited.

Reply:
swiftlint autocorrect should get all of em for u